snoop/data/digests.py

```diff
@@ -87,13 +88,27 @@ def gather(blob, collection_pk, **depends_on):
     with models.Blob.create() as writer:
         writer.write(json.dumps(rv).encode('utf-8'))
 
-    collection.digest_set.update_or_create(
+    digest, _ = collection.digest_set.update_or_create(
         blob=blob,
         defaults=dict(
             result=writer.blob,
         ),
     )
 
+    index.laterz(digest.pk)
```

Comment:
How can we create these tasks for existing digests? So we can reuse the digest blobs that have already been created.

Reply:
Rerunning all `digests.gather` tasks?

Reply:
Yeah, but that's i/o intensive, they each read a bunch of blobs, and write the output blob.

Reply:
Then I'll write you a one-liner to pass to django shell.

Reply:
Perfect, thanks!